static/js/common.js

```diff
@@ -0,0 +1,11 @@
+  // Fix bootstrap backdrop stacking when having multiple modals
+  $(document).on('show.bs.modal', '.modal', function() {
+    const zIndex = 1000 + Math.max(...Array.from(document.querySelectorAll('*')).map((el) => +el.style.zIndex));
```

Comment:
this is very slow,  takes 140ms on my side ;)
might wanna filter more via the selector, eg. taking only modal divs into account.

Reply:
Yeah, just removed it and used a static high zindex.